configmap-reload/Dockerfile

```diff
@@ -13,7 +13,7 @@ WORKDIR /go/src/github.com/jimmidyson/configmap-reload
 RUN curl -fsSL -o configmap-reload.tar.gz "https://github.com/jimmidyson/configmap-reload/archive/v${CONFIGMAP_RELOAD_VERSION}.tar.gz" \
     && tar -x -z --strip-components 1 -f configmap-reload.tar.gz \
     && rm -f configmap-reload.tar.gz \
-    && CGO_ENABLED=0 go install -ldflags="-w -s" ./
+    && CGO_ENABLED=0 go install --installsuffix cgo -ldflags="-w -s -extldflags '-static'" ./
```

Comment:
This modification is not required on recent golang but I don't request change because the original Makefile is written as it.